subprojects/gradle/lint-report/src/main/kotlin/com/avito/android/lint/LintReportTask.kt

```diff
@@ -45,10 +53,20 @@ abstract class LintReportTask : DefaultTask(), BuildVerdictTask {
         copyReports()
         val models = parseReports()
         sendToBitbucket(models)
+        if (slackReportChannel.isPresent) {
```

Comment:
Does it have the sense to run this task without a slack report?

Reply:
yep https://github.com/avito-tech/avito-android/pull/621#discussion_r519822366